src/Test.UnitTests.Sarif.Driver/Sdk/AnalyzeCommandBaseTests.cs

```diff
@@ -446,11 +539,9 @@ public void ExceptionRaisedInvokingInitialize()
         [Fact]
         public void FileUri()
         {
-            var uri = new Uri(this.GetType().Assembly.Location);
-
             var options = new TestAnalyzeOptions()
             {
-                TargetFileSpecifiers = new string[] { uri.ToString() },
+                TargetFileSpecifiers = new string[] { this.GetType().Assembly.Location },
```

Comment:
Don't know the original purpose of this test case. 
But before the change, 'uri.ToString()' will returl a string starting with "file:/". Do we still plan to support that, which is absolute file URI starting with file scheme prefix? If we don't plan to the support that? The essence of the fix is replacing 'uri.LocalPath' to 'uri.OriginalPath', that is it.

Reply:
I would say the idea is we always call into things that open files using variants that use OriginalString.  We know this is fine because someone passed us the string into a URI in an original form that came from... well .. something.  If that something is a Directory.GetFiles() call, then it's going to be in a format that the .NET APIs can handle for opening.  If that something is directly off the command line, then maybe someone will pass in file://whatever and an open call could fail.  So yes, we'd be dropping support for that very specific case to make the problem of handling percents in file paths more tractable.

Does anyone do that?  I doubt it.  So it seems like a favorable tradeoff.  We'll see if it passes CR. :)